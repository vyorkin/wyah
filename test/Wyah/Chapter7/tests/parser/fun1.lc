-- Functions 1

let const x y = x;
let compose f g = \x -> f (g x);
