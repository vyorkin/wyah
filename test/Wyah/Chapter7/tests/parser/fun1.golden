Program [Decl "compose" (ELam (Var "g") (ELam (Var "f") (ELam (Var "x") (EApp (EVar (Var "f")) (EApp (EVar (Var "g")) (EVar (Var "x"))))))),Decl "const" (ELam (Var "y") (ELam (Var "x") (EVar (Var "x"))))]